components/alerts/AlertsIntroPanel.vue

```diff
@@ -44,11 +62,29 @@ const emit = defineEmits(["dateRangeChanged"]);
           </p>
           <p class="italic inline-flex items-center whitespace-nowrap">
             {{ $t("ifYouAreZoomedOutAlertsWillBeShownAsA") }}&nbsp;
-            <img
+            <NuxtImg
```

Comment:
This refactor introduces a red background for the png.

![image](https://github.com/user-attachments/assets/a8d52922-5fbf-4ece-abeb-7f6ccc6ca8c8)

Not sure what the best way to fix it is (maybe getting rid of `preset="icon"`?), but if it results in a lot of bending over backwards, maybe best to just stick with a plain ol' `<img>` tag for this one as it's just a tiny PNG.